spec/unit/models/services/service_binding_spec.rb

```diff
@@ -637,13 +637,17 @@ def new_model
     describe '#destroy' do
       it 'cascade deletes all ServiceBindingOperations for this binding' do
```

Comment:
A more generic description would be better now, for instance "it cascades deletion of related dependencies"